app/assets/stylesheets/application.scss

```diff
@@ -56,7 +56,14 @@ $search-result: #008cba;
   .text-field {
     padding: 5px;
     height: 25px;
-    margin-bottom: 15px;
+    margin-bottom: 1px;
+  }
+
+  .error-explanation {
+    padding-bottom: 15px;
+    font-size: 10px;
+    font-style: italic;
+    color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.